lib/web/ui/integration.go

```diff
@@ -234,6 +234,29 @@ type AWSOIDCDeployDatabaseServiceResponse struct {
 	ClusterDashboardURL string `json:"clusterDashboardUrl"`
 }
 
+// AWSOIDCEnrollEKSClustersRequest is a request to ListEKSClusters using the AWS OIDC Integration.
+type AWSOIDCEnrollEKSClustersRequest struct {
+	Region             string   `json:"region"`
+	ClusterNames       []string `json:"clusterNames"`
+	JoinToken          string   `json:"joinToken"`
+	ResourceID         string   `json:"resourceId"`
+	EnableAppDiscovery bool     `json:"enableAppDiscovery"`
+	EnableAutoUpgrades bool     `json:"enableAutoUpgrades"`
```

Comment:
EnableAutoUpgrades is also known on the backend, accepting it from the frontend is less secure

Reply:
Also moved to the backend.